packages/react-native/package.json

```diff
@@ -112,7 +112,7 @@
   },
   "dependencies": {
     "@jest/create-cache-key-function": "^29.6.3",
-    "@react-native-mac/virtualized-lists": "0.76.0-main",
+    "@react-native-mac/virtualized-lists": "workspace:*",
     "@react-native/assets-registry": "0.76.0-main",
     "@react-native/codegen": "0.76.0-main",
     "@react-native/community-cli-plugin": "0.76.0-main",
```

Comment:
Could evening else also change to workspace:* now?

Reply:
I think so, but we don't publish `@react-native/community-cli-plugin` from here so it should probably be set to `workspace:^` instead.

Reply:
Probably a followup change.